sonar-dotnet-shared-library/src/main/java/org/sonar/plugins/dotnet/tests/Coverage.java

```diff
@@ -86,33 +87,12 @@ Map<Integer, Integer> hits(String file) {
   }
 
   List<BranchCoverage> getBranchCoverage(String file) {
-    List<BranchCoverage> result = new ArrayList<>();
-
-    Map<Integer, List<BranchPoint>> branchPointsPerLine = branchPoints.stream()
+    return branchPoints.stream()
       .filter(point -> point.getFilePath().equals(file))
-      .collect(Collectors.groupingBy(BranchPoint::getStartLine));
-
-    for (Map.Entry<Integer, List<BranchPoint>> lineBranchPoints : branchPointsPerLine.entrySet()){
-      if (lineBranchPoints.getValue().size() > 1){
-        int line = lineBranchPoints.getKey();
-
-        List<BranchPoint> linePoints = lineBranchPoints.getValue();
-        Map<String, List<BranchPoint>> groupsByKey = linePoints.stream().collect(Collectors.groupingBy(BranchPoint::getUniqueKey));
-
-        int coveredBranchPoints = 0;
-        for (Map.Entry<String, List<BranchPoint>> group: groupsByKey.entrySet()){
-          if (group.getValue().stream().filter(point -> point.getHits() >0).count() > 0){
-            coveredBranchPoints++;
-          }
-        }
-
-        if(groupsByKey.size() > 1) {
-          result.add(new BranchCoverage(line, groupsByKey.size(), coveredBranchPoints));
-        }
-      }
-    }
-
-    return result;
+      .collect(Collectors.groupingBy(BranchPoint::getStartLine)).entrySet().stream()
```

Comment:
Is startLine considered the line in the .cs file or IL?

Reply:
`.cs`